polygeist/tools/cgeist/driver.cc

```diff
@@ -710,6 +723,9 @@ static LogicalResult finalize(mlir::MLIRContext &Ctx,
     });
   }
 
+  if (!NoSYCLDeviceOnly)
```

Comment:
This is not intuitive. Can the option name be changed to `SYCLDeviceOnly` so that this code would read:
```
  if (SYCLDeviceOnly)
      eraseHostCode(*Module);